doc/vim-slime.txt

```diff
@@ -205,7 +206,22 @@ A pane relative to the currently active one in the target session
   - "up"/"down"/"right"/"left" for the pane in that direction relative to the location of the active pane
 
 ==============================================================================
-8. X11 Configuration 				*slime-x11*
+8. Wezterm Configuration 			*slime-wezterm*
+
+Wezterm is not the default, to use it you will have to add this line to your
+|.vimrc|:
+>
+	let g:slime_target = "kitty"
```

Comment:
```suggestion
	let g:slime_target = "wezterm"
```

copy-pasta?

Reply:
Haha, yes i used kitty as a base but must have missed changing this